src/kOS/Binding/BindingsUniverse.cs

```diff
@@ -81,7 +96,7 @@ public override void AddTo(SharedObjects shared)
                 {
                     TimeWarp.Modes toSet;
 
-                    switch (val.ToString())
+                    switch (val.ToString().ToUpper())
```

Comment:
this was an existing bug that i fixed, now:

    SET WARPMODE TO "PhYSIcs".

should work